src/pages/tickets/org/orgHeader/index.tsx

```diff
@@ -482,28 +527,37 @@ export const OrgHeader = ({
     <>
       <FillContainer>
         <Header>
-          <UrlButtonContainer data-testid="url-button-container">
-            {website !== '' ? (
-              <UrlButton onClick={() => handleWebsiteButton(website)}>
-                <img src={websiteIcon} alt="" />
-                Website
-              </UrlButton>
-            ) : (
-              ''
-            )}
-            {github !== '' ? (
-              <UrlButton onClick={() => handleGithubButton(github)}>
-                <img src={githubIcon} alt="" />
-                Github
-              </UrlButton>
-            ) : (
-              ''
-            )}
-          </UrlButtonContainer>
-          <Button onClick={handlePostBountyClick}>
-            <img src={addBounty} alt="" />
-            Post a Bounty
-          </Button>
+          <Leftheader>
+            <ImageContainer src={img} width="72px" height="72px" alt="organization icon" />
+            <CompanyNameAndLink>
+              <CompanyLabel>{name}</CompanyLabel>
+              <UrlButtonContainer data-testid="url-button-container">
+                {website !== '' ? (
+                  <UrlButton onClick={() => handleWebsiteButton(website)}>
+                    <img src={websiteIcon} alt="" />
+                    Website
+                  </UrlButton>
+                ) : (
+                  ''
+                )}
+                {github !== '' ? (
+                  <UrlButton onClick={() => handleGithubButton(github)}>
+                    <img src={githubIcon} alt="" />
+                    Github
+                  </UrlButton>
+                ) : (
+                  ''
+                )}
+              </UrlButtonContainer>
+            </CompanyNameAndLink>
+          </Leftheader>
+          <RightHeader>
+            <CompanyDescription>{description}</CompanyDescription>
```

Comment:
can we add a test to assert that this description shows up?

Reply:
There's no data for description in all the org data yet. Should I still include the test for it?

Reply:
Yes. The purpose of this pr is for the description of an org to be displayed. It is also stated in the acceptance criteria.

Reply:
Okay, I'll add the test to it and raise another PR. Thanks for the review